config/routes.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+
+  get "/users/:id", to: "users#show"
+  get "/users", to: "users#index"
+  get "/users/:id/posts", to: "posts#show"
+  get "/users/:id/posts/:post_id", to: "posts#show"
+
```

Comment:
- [ ] It would be nice if you add your routes in the following way which is easier, new, organized and correct :blush:

```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

```